pkg/crc/constants/constants.go

```diff
@@ -223,6 +221,18 @@ func GetDefaultMemory(preset crcpreset.Preset) int {
 	}
 }
 
+func GetDefaultPersistentVolumeSize(preset crcpreset.Preset) int {
+	switch preset {
+	case crcpreset.OpenShift, crcpreset.OKD:
+		return 3
+	case crcpreset.Microshift:
+		return 15
+	default:
+		// should not be reached
+		return 3
```

Comment:
right, we might have to increase the default disk-size, but there's very limited free space available, and currently that space is  shared for both image-registry and host container image storage, with this since we have to create a separate partition this becomes more apparent:

```
[core@crc ~]$ df -h /dev/vda4
Filesystem      Size  Used Avail Use% Mounted on
/dev/vda4        31G   23G  7.6G  76% /
```

We only have 7.6G free space, of which now `4.6G` is for host container images and `3G` for PVs

Reply:
maybe we can increase the default `disk-size` to 40 and default `persistent-volume-size` to 7, these are just arbitrary numbers, but it'd leave around 7gb free space on the root partition (closer to the current situation)

(the bundle sizes are also increasing which would mean less free space for workloads, right? i saw the `4.16.3` bundle from CI in 400MB larger than the 4.16.0 bundle)

Reply:
For what it's worth, increasing the guest disk size should not increase the bundle size nor the used disk space after `crc start` as long as the additional space is empty.